cmd/rekor-cli/app/verify.go

```diff
@@ -153,10 +154,17 @@ var verifyCmd = &cobra.Command{
 			}
 		}
 
-		if viper.IsSet("uuid") && (viper.GetString("uuid") != o.EntryUUID) {
-			return nil, fmt.Errorf("unexpected entry returned from rekor server")
+		if viper.IsSet("uuid") {
+			uuid, err := sharding.GetUUIDFromIDString(viper.GetString("uuid"))
+			if err != nil {
+				return nil, err
+			}
+			if uuid != o.EntryUUID {
+				return nil, fmt.Errorf("unexpected entry returned from rekor server")
+			}
 		}
 
+		// Note: the returned entry UUID is the UUID (not include the Tree ID)
```

Comment:
shouldn't the API return the full entryUUID (including the treeID)?

Reply:
That's my main question in the PR description. However, the behavior is mixed... e.g. `rekor-cli get` returns just the raw UUID as well, and the request API here also may take the raw UUID.

Originally I had changed the response payload to fix this. Should I go back to that?